package.json

```diff
@@ -12,8 +12,8 @@
     "test": "jest"
   },
   "dependencies": {
-    "@googlemaps/js-api-loader": "^1.12.8",
-    "@googlemaps/markerclusterer": "^1.0.11",
+    "@googlemaps/js-api-loader": "^1.12.10",
```

Comment:
why this is changes ? @doohanas

Reply:
Hello, kak @adzharamrullah. When I pull from the main, I cannot run this project then, I was asked to install `@googlemaps/js-api-loader` and `@googlemaps/markerclusterer`. So I just install using this command `npm run install --save googlemaps/js-api-loader @googlemaps/markerclusterer`. I don't know if this package is different version kak.

Reply:
okay thanks for your response may be we talk about this later